pkgs/applications/window-managers/i3/status.nix

```diff
@@ -9,15 +25,15 @@ stdenv.mkDerivation rec {
     sha256 = "0929chhvyq9hg4scpcz8r9zn3s9jvbg6a86k3wqa77qg85rh4kaw";
   };
 
-  nativeBuildInputs = [ meson ninja perl pkg-config asciidoc xmlto docbook_xml_dtd_45 docbook_xsl ];
-  buildInputs = [ libconfuse yajl alsa-lib libpulseaudio libnl ];
+  nativeBuildInputs = [meson ninja perl pkg-config asciidoc xmlto docbook_xml_dtd_45 docbook_xsl];
+  buildInputs = [libconfuse yajl alsa-lib libpulseaudio libnl];
 
   meta = {
     description = "Generates a status line for i3bar, dzen2, xmobar or lemonbar";
     homepage = "https://i3wm.org";
-    maintainers = [ ];
+    maintainers = [];
     license = lib.licenses.bsd3;
     platforms = lib.platforms.all;
+    mainProgram = "i3status";
```

Comment:
Changes other than this line are barely necessary.

Reply:
Auto-formatter worked a little too well on this one...